google/cloud/firestore_v1/async_collection.py

```diff
@@ -72,6 +72,10 @@ def _query(self) -> async_query.AsyncQuery:
         """
         return async_query.AsyncQuery(self)
 
+    async def chunkify(self, chunk_size: int):
```

Comment:
It certainly could be. It seems like a helpful utility, but I would also be fine making it internal.

Reply:
If we have an example for how users would use it we could expose it, but generally I think we prefer to keep things internal if we aren't sure since it is a commitment to future design.